pbnh/tests/paste_curl_test.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/python3
+import pycurl
+import unittest
+import json
+import hashlib
+import tempfile
+import os
+
+from pbnh import conf
+from pbnh import app
+
+config = conf.get_config().get('server')
+host = config.get('bind_ip')
+port = config.get('bind_port')
+URL = 'http://{}:{}/'.format(host, port)
+
+
+class TestPost(unittest.TestCase):
+    def setUp(self):
+        self.c = pycurl.Curl()
+        self.c.setopt(pycurl.URL, URL)
+        self.db_fd, app.app.config['DATABASE'] = tempfile.mkstemp()
```

Comment:
Not explicitly creating it will only work for sqlite, so if similar tests are eventually created using a different backend (though I'm not sure they would need to me) they would fail. I'm also not sure how much I trust sqlite to properly dynamically allocate columns on insertion.

Additionally, the zen of python says "Explicit is better than implicit", which I fully believe in.